metadata-ingestion/src/datahub/ingestion/source/mlflow.py

```diff
@@ -115,6 +116,13 @@ class MLflowConfig(StatefulIngestionConfigBase, EnvConfigMixin):
         default=None, description="Mapping of source type to datahub platform"
     )
 
+    username: Optional[str] = Field(
+        default=None, description="Username for MLflow authentication"
+    )
+    password: Optional[str] = Field(
+        default=None, description="Password for MLflow authentication"
+    )
```

Comment:
throw a validation error if one is set but not both

Reply:
done